packages/backend/src/apps/m365-excel/auth/refresh-token.ts

```diff
@@ -0,0 +1,8 @@
+import type { IAuth } from '@plumber/types'
+
+const refreshToken: NonNullable<IAuth['refreshToken']> = async function (_$) {
+  // This is a no-op, because our beforeRequest.addAuthToken will automatically
+  // refresh the token when our response interceptor retries from a 401.
+}
+
+export default refreshToken
```

Comment:
oh actually can i ask is this needed? cus i see its an optional field in IBaseAuth

Reply:
As discussed - this is for auto retry in our response interceptor